### ISSUE TITLE
AudioHandler constructor specifications

### ISSUE THREAD

**Issue by agustafson19** [JEAB186X/Project#4]
The Audio Handler will need to have an instance of the record handler passed thereto:

public AudioHandler(RecordHandler rh);

This will allow the current record to be loaded so that it can be played.

In the future, the RecordHandler will keep track of what instrument will be played, so you can get that information therefrom as well.

**Comment by kelvis24** [comment]
Yeah, Thanks Aarvid. I noticed that yesterday and have already created the method stub for it in the audioHandler Class.
I really dont understand what rh.method() will return a file to be read as the record. I planned on asking you about it briefly at the start of tomorrows class.
